source/validate_cfg.cpp

```diff
@@ -290,13 +295,12 @@ spv_result_t StructuredSwitchChecks(const ValidationState_t& _,
         // must immediately precede T2 in the list of OpSwitch Target operands.
         if ((switch_inst->operands().size() < j + 2) ||
             (case_fall_through != switch_inst->GetOperandAs<uint32_t>(j + 2))) {
-          return _.diag(SPV_ERROR_INVALID_CFG)
+          return _.diag(SPV_ERROR_INVALID_CFG, -1)
```

Comment:
The OpSwitch if its accessible would probably be the right thing to disassemble here.

Reply:
Nice, now they all give diagnostic again. Thanks.